static/js/analytics.js

```diff
@@ -130,9 +130,9 @@ function setSurveyedCookie() {
 function analyticsSurveyLogic() {
   const csatWrapperEl = document.querySelector(".js-csat-wrapper");
   if (!isGoogleAnalyticsAvailable()) {
-    csatWrapperEl.remove();
     return;
   }
+  csatWrapperEl.classList.add("is-visible");
```

Comment:
Issue: I ran into a bug while testing another PR and added a fix in this commit: https://github.com/mozilla/fx-private-relay/pull/1545/commits/98157cb2f59f17c4c366fa0023ee9b993b2f38c5

Suggestion: We will need to port this over/resolve a merge conflict at some point soon. 

```suggestion
  csatWrapperEl?.classList.add("is-visible");
```

Reply:
That should be fairly easy to resolve, no? 5de94304